pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -318,6 +310,12 @@ export class SETroubleshootPage extends React.Component {
         e.stopPropagation();
     }
 
+    onSelect(_, isSelected, rowId) {
+        const selected = Object.assign(this.state.selected);
+        selected[this.props.entries[rowId / 2].key] = isSelected;
```

Comment:
Divisions in an array index, and doing arithmetic with an ID both look weird..  That could at least use a comment. There is somehow two rows for an entry? Can we be sure the ID is even? Can we even be sure that it contains a number? (type-wise, it's a string, no?) Could it be `//` to make sure it uses integer division, or perhaps avoid arithmetic at all?

Reply:
There is a row also for the expandable items. Therefore it's tr = number of rows * 2.

onSelect returns an index, not an DOM id. That's a confusing name for this variable indeed. 

For avoiding arithmetic at all I need to rewrite cockpit-components-table.jsx to use the new PF4 table (this uses an deprecated PF4 table) which I want to do - but out of the scope of this PR.

Reply:
Thanks for the explanation -- so this will disappear eventually, good.